### PR TITLE
Adding a new test to run 2 ways netperf workload.

### PR DESCRIPTION
**Pull request by ahaware**
This will enable to run netperf workload with run.sh

### CODE REVIEW

File: config/np_network_internode/netperf_client_pod.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        memory: "4200Mi"
+        cpu: "16000m"
+      requests:
+        memory: "4200Mi"
+        cpu: "1000m"
+        ephemeral-storage: "5Gi"
+    command: ["/bin/sh","-c"]
+    args: 
+      - sleep 2s;
+        apt-get update;
```
Comment (ganesank-git) [inline config/np_network_internode/netperf_client_pod.yaml:35]: We can remove these iperf/qperf related installations.

File: config/np_network_internode/README [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Run dp_network_internode individually using the following command:
+cd ../../; ./run.sh -t "np_network_internode" -r <run-tag> -o <output-dir>
```
Comment (ganesank-git) [inline config/np_network_internode/README:2]: Lets call this dp_netperf_internode to keep it consistent with respect to naming the data plane and control plane workloads.

File: config/np_network_internode/netperf_client_pod.yaml (hunk 1 above)
Comment (ahaware) [reply config/np_network_internode/netperf_client_pod.yaml]: Done.

File: config/np_network_internode/README (hunk 2 above)
Comment (ahaware) [reply config/np_network_internode/README]: Done.

**Pull request merged**